### ISSUE TITLE
[java] NoPackage: False Negative for enums

### ISSUE THREAD

**Issue by adangel** [pmd/pmd#1782]
**Rule:** [NoPackage](https://pmd.github.io/pmd-6.13.0/pmd_rules_java_codestyle.html#nopackage)

**Description:**
Enums within the default package are not detected.

**Code Sample demonstrating the issue:**

```
public enum Foo { BAR; }
```

This is fixed in 6.14.0 - see #1771

**Comment by adangel** [comment]
Annotations as well...
